lib/v0_bindings_extensions.rb

```diff
@@ -83,11 +83,21 @@ def self.create_from_query_result(query_result)
 
   Api::V0::Bindings::GetHighlightsParameters.class_exec do
     def query(user_id:)
-      highlights = ::Highlight.by_user(user_id)
+      users = []
+
+      if sets.blank? or sets.include?('user:me')
+        users.push(user_id)
+      end
+      if  sets.to_a.include?('curated:openstax') and scope_id.present?
+        curator_scope = CuratorScope.find_by(:scope_id => scope_id)
+        users.push(curator_scope.curator_id) unless curator_scope.nil?
```

Comment:
Should there be an error if the requested curator doesn't exist?

Reply:
no, we want to return an empty set if there is no curation, as an abstraction for the client